lib/index.ts

```diff
@@ -12,5 +12,5 @@ if (!WebAssembly.instantiateStreaming) {
 }
 
 export type { LncConfig, CredentialStore } from './types/lnc';
-
+export { lnrpc, looprpc, poolrpc, frdrpc } from '@lightninglabs/lnc-core';
```

Comment:
The LND subserver RPCs are missing from this list. Why not use `export *` instead to expose everything from [lnc-core](https://github.com/lightninglabs/lnc-core/blob/main/lib/index.ts)? I can't think of a downside to doing it this way.